src/chronometer.js

```diff
@@ -1,33 +1,54 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback) {
+        printTimeCallback();
+      }
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    
+      return Math.floor(this.currentTime / 60);
+
   }
 
   getSeconds() {
-    // ... your code goes here
+    
+    return this.currentTime % 60;
+
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    
+    if (value.toString().length === 2) {
+      return value.toString()
+    } else if (value.toString().length < 2) {
+        return "0"+value.toString()
+    }
   }
 
   stop() {
-    // ... your code goes here
+
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    //let minutes = this.computeTwoDigitNumber(this.getMinutes);
```

Comment:
Esto estaba literalmente perfecto!!! lo unico que te faltaba es, que el this.getMinutes es una funcion(metodo) es decir le estas pasando como parametro no lo que devuelve ese metodo, si no el metodo en si, por eso lo unico que te falta aqui es ejecutar ese metodo es decir.... los parentesis jajajaj pero vamos que esta genial!!!! lo de abajo esta bien, pero esto esta mejor!!! BUEN TRABAJO!!!